release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -769,6 +765,10 @@ Netlink messages dropped by OVS vSwitch daemon due to netlink socket buffer over
 Netlink messages dropped by OVS kernel module due to netlink socket buffer overflow. This will result in packet loss.
 ----
 
+[id="ocp-4-13-network-observability-1_1-1_2"]
+==== Network Observability Operator
+The Network Observability Operator releases updates independently from the {product-title} minor version release stream. Updates are available through a single, rolling stream which is supported on all currently supported versions of {product-title} 4. Information regarding new features, enhancements, and bug fixes for the Network Observability Operator can be found in the xref:../networking/network_observability/network-observability-operator-release-notes.adoc[Network Observability release notes].
```

Comment:
```suggestion
The Network Observability Operator releases updates independently from the {product-title} minor version release stream. Updates are available through a single, rolling stream, which is supported on all currently supported versions of {product-title} 4. Information regarding new features, enhancements, and bug fixes for the Network Observability Operator can be found in the xref:../networking/network_observability/network-observability-operator-release-notes.adoc[Network Observability release notes].
```
I think this is a non-restrictive clause. If so, see the ISG: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-with-nonrestrictive-clauses-and-phrases